bin/build_docs.py

```diff
@@ -76,6 +76,7 @@ def main():
             "sample-gherkindoc-glossary",
             "--parser",
             "pytest_bdd",
+            "--raw-descriptions",
```

Comment:
This is kinda confusing as it sort of implies that the formatting will be like using back-ticks in git hub comments, or the noformat tag in JIRA.
Not sure I have a better idea, how about rst-descriptions or pass-thru-descriptions.
Not holding up the PR because of this but wanted to spark some discussion.